src/deposit.cairo

```diff
@@ -51,17 +69,18 @@ mod Deposit {
         decimals_difference: DecimalScale,
         total_borrowed: TokenAmount,
         borrow_const: u8
```

Comment:
Similar thing for this one. `borrow_const` has very little meaning. Suggest that you chnage this to something like `borrow_portion_percent`